manage-operational-layers/src/main/java/com/esri/arcgismaps/sample/manageoperationallayers/screens/LayersList.kt

```diff
@@ -0,0 +1,160 @@
+/* Copyright 2023 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.manageoperationallayers.screens
+
+import android.content.res.Configuration
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material.icons.filled.KeyboardArrowUp
+import androidx.compose.material3.Card
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.arcgismaps.mapping.layers.Layer
+import com.esri.arcgismaps.sample.manageoperationallayers.R
+
+/**
+ * Layout to display a list of operational layers on the map using [layerNames].
+ */
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun LayersList(
+    layerNames: List<String>,
+    inactiveLayers: List<Layer>,
+    onMoveLayerUp: (String) -> Unit = {},
+    onMoveLayerDown: (String) -> Unit = {},
+    onRemoveLayer: (String) -> Unit = {},
+    onAddLayer: (String) -> Unit = {},
+) {
+    Column {
+        Text(
+            modifier = Modifier.padding(12.dp),
+            text = "Active layers"
+        )
+        LazyColumn(modifier = Modifier.padding(12.dp)) {
+            items(layerNames.size, key = { layerNames[it] }) { index ->
```

Comment:
Hmm I feel like this is a flipped around from how it should be. I'm not sure what it's like on other platforms but I feel like the layer that's displaying on top should be at the top of this items list, not at the bottom.

Reply:
The list displayed corresponds with the order of the layers added to the map's operational layers:
```kotlin
// add the layers to the map's operational layers
mapViewState.arcGISMap.apply {
    operationalLayers.addAll(
        listOf(
            imageLayerElevation,
            imageLayerCensus,
            imageLayerDamage
        )
    )
}
```

This is also consistent with the order used in the iOS sample: 
<img src="https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/93d2a215-dc21-4b84-85ff-51ece76e6a47" width="250">

<img src="https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/8164b955-4a6a-4050-b45c-53161648d409" width="250">

Reply:
Cool, makes sense to be consistent with the other platforms